src/permutation/permutation.rs

```diff
@@ -966,7 +964,7 @@ mod test {
 
         let domain =
             GeneralEvaluationDomain::<E::Fr>::new(num_wire_mappings).unwrap();
-        let w = util::get_domain_attrs(&domain, "group_gen");
+        let w = domain.group_gen();
```

Comment:
I would prefer to avoid touching anything related to domain internal field obtentions until https://github.com/arkworks-rs/algebra/pull/333 is released.

This will make it much more easy and organic. But for now I would prefer to get all of the elements on the same way. Instead of some on one way, and some on the other..

Reply:
Not sure what you mean by "some one way and some on the other". This PR removed the entire `util::get_domain_attrs` method.